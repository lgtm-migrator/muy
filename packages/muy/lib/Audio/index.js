import React from "react"
import Box from "@material-ui/core/Box"

const Audio = ({ controls, display, height, ...props }) => (
  <Box
    component={"audio"}
    display={controls ? display : "none"}
    height={controls ? height : 0}
    {...props}
  />
)

export default Audio

Audio.defaultProps = {
  display: "inline-block",
}
