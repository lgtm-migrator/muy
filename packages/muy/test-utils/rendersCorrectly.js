import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
