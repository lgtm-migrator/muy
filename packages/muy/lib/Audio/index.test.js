import React from "react"
import Audio from "."
import rendersCorrectly from "../../test-utils/rendersCorrectly"

describe("<Audio /> component", () => {
  it("renders correctly", () => {
    rendersCorrectly(<Audio />)
  })
})
