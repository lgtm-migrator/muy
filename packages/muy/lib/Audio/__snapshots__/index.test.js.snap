// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Audio /> component renders correctly 1`] = `
<audio
  className="MuiBox-root MuiBox-root-2"
/>
`;
